theme/src/primer-nav.yml

```diff
@@ -10,6 +10,8 @@
       url: https://primer.style/css/tools/prototyping
     - title: Presentations
       url: https://primer.style/presentations
+    - title: CLI
```

Comment:
Do we want the title of this link to be `CLI` or `Command line`?

Reply:
🤔 🤔 🤔 Good question. I think CLI is fine but maybe others have other thoughts :)

Reply:
cc @ampinsk